### ISSUE TITLE
helm install fails with unable to recognize ""

### ISSUE THREAD

**Issue by lars-tice** [confluentinc/cp-helm-charts#357]
Hello,

Running minikube with kubernetes 1.16.3-00 & helm 3.0.0.  When I run:
```
helm repo add confluentinc https://confluentinc.github.io/cp-helm-charts/
helm repo update
helm install confluentinc/cp-helm-charts --name my-confluent-oss
```

I get the following error:
Error: validation failed: [unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2", unable to recognize "": no matches for kind "StatefulSet" in version "apps/v1beta1"]

It looks like commit #326, should've addressed this.  But a point of confusion for me is the part of the message that says: unable to recognize ""

Thanks

**Comment by felixk** [comment]
I get the same problem. Were you able to resolve?

**Comment by gAmUssA** [comment]
I don't believe we ever tested these charts with helm 3. @lars-tice did you have a chance to find solution?
